pkg/watch/mux.go

```diff
@@ -149,6 +151,7 @@ func (m *Broadcaster) Action(action EventType, obj runtime.Object) {
 // You must not call Action after calling Shutdown.
```

Comment:
Also, say that it blocks until all events have been distributed to all extant watchers.